fe/fe-core/src/main/java/org/apache/doris/datasource/jdbc/client/JdbcClient.java

```diff
@@ -221,7 +221,7 @@ public List<String> getDatabaseNameList() {
         Statement stmt = null;
         ResultSet rs = null;
         if (isOnlySpecifiedDatabase && includeDatabaseMap.isEmpty() && excludeDatabaseMap.isEmpty()) {
-            return getSpecifiedDatabase(conn);
+            return filterDatabaseNames(getSpecifiedDatabase(conn));
```

Comment:
You should also apply this modification where you override getDatabaseNameList()

Reply:
ok, it’s my mistake, i will modify in tonight